DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: DataTools
 Type: Package
 Title: Data Tools for Shiny Apps
-Version: 23.04.2.3
+Version: 23.04.2.4
```

Comment:
Please update to version 23.04.3 since this is a new feature.
The forth position I only use to trigger updates or to fix bugs.

Can you add a new entry  at the top in NEWS.md explaining what was changes? E.g.:

```
## Version 23.04.3

## New Features
- new functions to reach the isomemo API that were extracted from the iso-app
```

Usually I add exactly that part into the description of the open PR: https://github.com/Pandora-IsoMemo/data-tools/pull/38#issue-1686483324